docs/HowTo/write/design_considerations.md

```diff
@@ -22,15 +22,15 @@ Refer to the conceptual documentation if you're not familiar with:
 
 [`Circuits`](../../Concepts/circuits.md) are programmable but can't (practically and efficiently) prove any algorithm. The way constraints are represented make some things more natural ("snark-friendly") to do than others.
 
-Numbers used in constraints are not integers or floats, but finite field elements (big numbers modulo a prime $p$).
+Numbers used in constraints are not integers or floats, but finite field elements (big numbers modulo a prime `p`).
```

Comment:
previous version was actually correct, didn't render because the new doc didn't support Latex, fixed now 👍